HIRS_ProvisionerTPM2/docs/dependencies-centos.md

```diff
@@ -13,5 +13,4 @@ Please look up their respective names in the CentOS repositories.
 | protobuf   | 2.5.0        | 2.4.1 (estimated) | CentOS 7 base         | https://github.com/google/protobuf          |
 | re2        | 20160401     | 20160201          | CentOS 7 epel-release | https://github.com/google/re2               |
 | tpm2-tss   | 1.2.0        | 1.0.0             | CentOS 7 base         | https://github.com/intel/tpm2-tss           |
-| lshw       | B.02.18-12   | B.02.18-12        | CentOS 7 base         | https://github.com/lyonel/lshw              |
```

Comment:
We'll need to remember to remove this dependency from the different Guides in the Wiki when this gets merged.

Reply:
Actually for that matter does paccor need added to this list?

Reply:
I will add it to this file.

Reply:
I would say Paccor is fine here regardless of the discussion regarding the Package Requirements.